cli_docs/generate_cli_docs.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import sys
+
+import click
+from pkg_resources import iter_entry_points
+
+
+def generate_docs(name, filename):
+    console_scripts = [ep for ep in iter_entry_points('console_scripts', name=name)]
+
+    if len(console_scripts) == 0:
+        raise click.ClickException('"{}" is not an installed console script.'.format(name))
+
+    # read all lines from original file
+    original_lines = []
+    with open(filename, "r") as md_file:
+        original_lines = md_file.readlines()
```

Comment:
I think it is better to make some kind of variable and print it to file (not to make files that are partially editable sometimes)
In python one can simply do that with:
```
HEADER="""
#Line1
Line2
Line 4
"""
```